matrix/lambdas/daemons/mapper.py

```diff
@@ -46,8 +48,7 @@ def run(self, bundle_uuid: str, bundle_version: str):
         print(f"{self.request_id} Invoking {len(worker_chunk_specs)} worker lambdas...")
         for worker_chunk_spec in worker_chunk_specs:
             self.lambda_handler.invoke(LambdaName.WORKER, self._get_worker_payload(worker_chunk_spec))
-            print(f"{self.request_id} Worker invoked: start row "
-                  f"{worker_chunk_spec['start_row']}, num rows {worker_chunk_spec['num_rows']}")
+            print(f"{self.request_id} Worker invoked {worker_chunk_spec}")
```

Comment:
Wait, it looks like this still invokes 1 Worker for every chunk. See *

Reply:
Each `worker_chunk_spec` should now be a list of dicts rather than a dict. Is there a way to make that clearer? Maybe call it `worker_chunks_spec`?